schemas/BaseElements.xsd

```diff
@@ -2709,6 +2709,11 @@ http://www.epa.gov/compliance/monitoring/programs/caa/woodheaters.html</xs:docum
 			<xs:extension base="HVACSystemInfo">
 				<xs:sequence minOccurs="0">
 					<xs:element name="HeatPumpType" type="HeatPumpType" minOccurs="0"/>
+					<xs:element minOccurs="0" name="ColdClimate" type="xs:boolean">
+						<xs:annotation>
+							<xs:documentation>Is this a cold climate heat pump?</xs:documentation>
```

Comment:
My understanding is that "cold climate" heat pumps have inverter/variable
speed drives and that is what gives them curves that are better in cold
climates.  Maybe we should add a boolean for that as a specific technical
specification instead of the subjective "cold climate" designation.

On Mon, Nov 30, 2015 at 1:30 PM, Leo Rainer <notifications@github.com>
wrote:

> In schemas/BaseElements.xsd
> <https://github.com/hpxmlwg/hpxml/pull/58#discussion_r46182204>:
>
> > @@ -2709,6 +2709,11 @@ http://www.epa.gov/compliance/monitoring/programs/caa/woodheaters.html</xs:docum
> >  			<xs:extension base="HVACSystemInfo">
> >  				<xs:sequence minOccurs="0">
> >  					<xs:element name="HeatPumpType" type="HeatPumpType" minOccurs="0"/>
> > +					<xs:element minOccurs="0" name="ColdClimate" type="xs:boolean">
> > +						<xs:annotation>
> > +							<xs:documentation>Is this a cold climate heat pump?</xs:documentation>
>
> @nmerket <https://github.com/nmerket> @gmdickey
> <https://github.com/gmdickey> Glenn and I have discussed this before - it
> is problematic as there is no official designation, just performance curves
> that are better at lower temperature.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hpxmlwg/hpxml/pull/58/files#r46182204>.
>